openstack/client.go

```diff
@@ -197,6 +197,19 @@ func NewIdentityV3(client *gophercloud.ProviderClient) *gophercloud.ServiceClien
 	}
 }
 
+func NewIdentityAdmin(client *gophercloud.ProviderClient) (*gophercloud.ServiceClient, error) {
+	eo := gophercloud.EndpointOpts{
```

Comment:
@jrperritt Yes, it reuses the same `EndpointLocator` procedure as other resources.